speaker/bgp_controller.go

```diff
@@ -153,12 +153,12 @@ func (c *bgpController) ShouldAnnounce(l log.Logger, name string, _ []net.IP, po
 	}
 
 	if k8snodes.IsNetworkUnavailable(nodes[c.myNode]) {
-		level.Debug(l).Log("event", "skipping should announce bgp", "service", name, "reason", "speaker's node has NodeNetworkUnavailable condition")
+		level.Warn(l).Log("event", "skipping should announce bgp", "service", name, "reason", "speaker's node has NodeNetworkUnavailable condition")
 		return "nodeNetworkUnavailable"
 	}
 
 	if k8snodes.IsNodeExcludedFromBalancers(nodes[c.myNode]) {
-		level.Debug(l).Log("event", "skipping should announce bgp", "service", name, "reason", "speaker's node has labeled 'node.kubernetes.io/exclude-from-external-load-balancers'")
+		level.Warn(l).Log("event", "skipping should announce bgp", "service", name, "reason", "speaker's node has labeled 'node.kubernetes.io/exclude-from-external-load-balancers'")
```

Comment:
I don't think we want this to be warn either, as in regular scenarios, where the label is present, this might spam the logs.
Thoughts @oribon ?

Reply:
This is the one in particular that, if it had been a warning instead of a debug message, would have saved me *hours* of troubleshooting. Nowhere in the MetalLB docs is this label mentioned even once, nor the effect it has.

If users don't like seeing this warning message in their logs, they can manually use other labelSelectors to limit the node selection and stop the messages. But for folks like me who had no idea the `exclude-from-external-load-balancers` label even *existed*, I strongly feel like this should not be hidden behind a debug log.

Reply:
Or, when #2280 lands, I'll just enable that option and be happy. :)

Reply:
https://github.com/metallb/metallb/pull/2280 is gonna land soon and we'll release it.
I added this as part of the troubleshooting guide, and pinned an issue.
Hope it's enough, but if not a user should enable debug logs

Reply:
Not sure I understand the "spam the logs" concern, as if the user has a speaker on that node they probably expect it to be able to announce lbs and would want to unlabel the node asap, but I'm fine with either approach (hoping the docs are enough 😅)